src/stylesheets/components/_navigation.scss

```
@@ -20,13 +15,17 @@ $navigation-height: 50px;
 }
 
 .app-navigation__list-item {
+  $navigation-height: 50px;
```

Should this use `govuk-spacing`?
```suggestion
  $navigation-height: govuk-spacing(8);
```

---

Good question! I don't think so, because this doesn't really have anything to do with spacing and isn't really tied to the spacing scale.

We had this sort of thing before where people used `$gutter-width` whenever they wanted something to be 30px, even though it had nothing to the width of a gutter. So I generally try and avoid tying to a variable unless the explicit intent is to tie the two together.

What do you think?

---

That makes sense to me. If we decided to change the spacing scale, having this tied to it wouldn't really make a difference (or rather we would need to probably update this anyway). Resolving.